lib/kubernetes-deploy/deferred_summary_logging.rb

```diff
@@ -34,12 +34,16 @@ def heading(text, secondary_msg = '', secondary_msg_color = :cyan)
     end
 
     # Outputs the deferred summary information saved via @logger.summary.add_action and @logger.summary.add_paragraph
-    def print_summary(success)
-      if success
-        heading("Result: ", "SUCCESS", :green)
+    def print_summary(status)
+      status_string = status.to_s.humanize.upcase
+      if status == :success
+        heading("Result: ", status_string, :green)
         level = :info
+      elsif status == :timed_out
+        heading("Result: ", status_string, :yellow)
+        level = :warn
```

Comment:
I think we still want `:fatal` for this, since it did in fact kill the deploy... but also because `error` and `warn` with this particular logger make the whole lined coloured:

![image](https://user-images.githubusercontent.com/4789493/38001776-b964b3d4-31fc-11e8-9755-b81876076847.png)